spring-cloud-starter-stream-source-sftp-dataflow/README.adoc

```diff
@@ -0,0 +1,130 @@
+//tag::ref-doc[]
+= SFTP Data Flow Source
+
+This is a specialized version of the `sftp` source which always outputs a Data Flow Task Launch Request as JSON,
+compatible with the https://github.com/spring-cloud-stream-app-starters/tasklauncher-data-flow[tasklauncher-data-flow
+] sink.
```

Comment:
Consider to follow a "Sentence per line" rule: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

That's going to be much easier to review and maintain in the future.